### PR TITLE
image_builder: fix `blob` command

### PR DESCRIPTION
**Pull request by nalajcie**
## Description

- image_builder: fix `blob` command
- image_builder: small typing/linter fixes

## Motivation and Context
JIRA: CI-494

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (refactoring, style fixes, git/CI config, submodule management, no code logic changes)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [x] New test added: https://github.com/phoenix-rtos/phoenix-rtos-project/pull/1211
- [x] Tested by hand on: `zynq7000-qemu`.

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing linter checks and tests passed.
- [ ] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

File: scripts/image_builder.py [hunk 1]
```diff
@@ -325,19 +325,32 @@ def __post_init__(self, extra_flags: str = '', filename_args: str = ''):
...
         # filename can be either relative to PROG_STRIPPED or absolute (in ROOTFS)
         self.filename, self.abspath = self._resolve_filename(self.filename)
 
         self.size = os.path.getsize(self.abspath)
 
-        # TODO: add parsing maps?
-        for req_val_name in ("filename", "text_map", "data_maps"):
+        # HACKISH: blob cmd - treat 'text_map' as data map to support specifying by string
+        if self.name == "blob" and not self.data_maps and self.text_map:
+            print(asdict(self))
```
Comment (ziemleszcz) [inline scripts/image_builder.py:338]: Debug?

**Review by ziemleszcz — DISMISSED** [review]
Looks OK except debug print and one extra space before data map.

File: scripts/image_builder.py (hunk 1 above)
Comment (nalajcie) [reply scripts/image_builder.py]: yep, sorry - removed

**Comment by nalajcie** [comment]
> Looks OK except debug print and one extra space before data map.

I've noticed the extra space  but couldn't think of not over-complicated way to remove it (and `plo` didn't have problem).

I've proposed some solution (with extra comment to avoid removing it in the future).

**Review by ziemleszcz — APPROVED** [review]

**Pull request merged**